refresh-collaborators.py

```diff
@@ -0,0 +1,44 @@
+import os
+from bs4 import BeautifulSoup
+from github import Github
+import yaml
+
+### GET THE NAMES OF THE KAFKA COMMITTERS FROM THE apache/kafka-site REPO ###
+github_token = os.environ.get('GITHUB_TOKEN')
+g = Github(github_token)
+repo = g.get_repo("apache/kafka-site")
+contents = repo.get_contents("committers.html")
+content = contents.decoded_content
+soup = BeautifulSoup(content, "html.parser")
+committer_logins = [login.text for login in soup.find_all('div', class_='github_login')]
+
+### GET THE CONTRIBUTORS OF THE apache/kafka REPO ###
+n = 10
+repo = g.get_repo("apache/kafka")
+contributors = repo.get_contributors()
```

Comment:
I'm worried about taking the top ten contributors by _lifetime_ commits, rather than by commits from the last year (as in `git shortlog --email --numbered --summary --since=2022-04-28`), since we have some prolific contributors in the past who are no longer active.

Reply:
Hello @vvcephei, I understand your concern. I will make changes to account for this.